packages/core/src/components/ButtonGroup/index.tsx

```diff
@@ -25,7 +28,17 @@ export default function ButtonGroup({ children, endAlign, stacked }: Props) {
     >
       {React.Children.map(children, child =>
         child ? (
-          <div className={cx(stacked ? styles.cell_stacked : styles.cell)}>{child}</div>
+          <div
+            className={cx(
+              styles.cell,
+              stacked && styles.cell_stacked,
+              stretched && styles.cell_block,
+            )}
+          >
+            {React.cloneElement(child as React.ReactElement<ButtonOrLinkProps>, {
+              block: stretched,
```

Comment:
Can we not force block via flexbox?

Reply:
we could if i targeted `button` or `a` or `div` or `span` direct child of the cell. happy to go that route too, if you want

Reply:
Worth a shot. I'm not a fan of cloning elements.

Reply:
sounds good !